include/z64math.h

```diff
@@ -87,12 +87,26 @@ typedef struct {
     /* 0x0C */ Vec3f b;
 } LineSegment; // size = 0x18
 
-// Defines a point in the spherical coordinate system
 typedef struct {
-    /* 0x0 */ f32 r;       // radius
-    /* 0x4 */ s16 pitch;   // polar (zenith) angle
-    /* 0x6 */ s16 yaw;     // azimuthal angle
-} VecSph; // size = 0x8
+    /* 0x0 */ f32 r; // radius
+    /* 0x4 */ s16 pitch; // depends on coordinate system. See below.
+    /* 0x6 */ s16 yaw; // azimuthal angle
+} VecSphGeo; // size = 0x8
+
+// Defines a point in the spherical coordinate system.
+// Pitch is 0 along the positive y-axis (up)
+typedef VecSphGeo VecSph;
+
+// Defines a point in the geographic coordinate system.
+// Pitch is 0 along the xz-plane (horizon)
+typedef VecSphGeo VecGeo;
```

Comment:
They have to be the same struct because of:
```c
VecGeo* OLib_Vec3fToVecGeo(VecGeo* dest, Vec3f* vec) {
    VecSph sph;

    OLib_Vec3fToVecSph(&sph, vec);
    sph.pitch = 0x4000 - sph.pitch;

    *dest = sph;

    return dest;
}
```
Where `sph` is used for both geo and sph coordinates.

If we want to rename `pitch` -> `latitude` or `altitude`, then that should be a global-wide change and would need a lot of discussion. But currently, pitch is the word used for that terminology

Reply:
Same with renaming `yaw` -> `azimuth`, although I don't see that being a popular rename